### PR TITLE
Closes #39 - Update timer and stopwatch page

### LINKED ISSUE #39: Update timer and stopwatch page

### PR DESCRIPTION
**Pull request by andriy-korchynskyy**
image
image
image
image

### CODE REVIEW

**Comment by roman-fay** [comment]
what I can see right away - position of view switch (circle / line) should be unified as well

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (roman-fay) [inline .vscode/launch.json]: should this be in repository? probably not

File: src/app/line-progress/line-progress.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+@Component({
+  selector: 'app-line-progress',
+  templateUrl: './line-progress.component.html',
+  styleUrls: ['./line-progress.component.css']
+})
+export class LineProgressComponent implements OnInit {
+  @Input() value: number = 0;
+  @Input() isRun: boolean;
+  @Input() isPause: boolean;
```
Comment (roman-fay) [inline src/app/line-progress/line-progress.component.ts]: "isPaused"

File: src/app/line-progress/line-progress.component.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-line-progress',
+  templateUrl: './line-progress.component.html',
+  styleUrls: ['./line-progress.component.css']
+})
+export class LineProgressComponent implements OnInit {
+  @Input() value: number = 0;
+  @Input() isRun: boolean;
```
Comment (roman-fay) [inline src/app/line-progress/line-progress.component.ts]: "isActive" (if I understand intent right)

File: src/app/services/stopwatch.service.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        }
+      }
+    
+      updateTime() {
+    
+        if(this.ticks > 24*3600){
+          this.pause();
+        }
+        else{
+          this.hour = Math.floor( this.ticks / 3600 );
```
Comment (roman-fay) [inline src/app/services/stopwatch.service.ts]: magic number

Comment (roman-fay) [reply src/app/services/stopwatch.service.ts]: a lot of them :)

File: src/app/stopwatch/stopwatch.component.css [hunk 5]
```diff
@@ -11,55 +11,55 @@
...
-.run-stopwatch{
-    color: blue;
+::ng-deep .mat-slide-toggle.mat-checked:not(.mat-disabled) .mat-slide-toggle-thumb {
+    background-color: blue; /*replace with your color*/
 }
 
-.pause-stopwatch{
-    color: red;
+::ng-deep .mat-slide-toggle.mat-checked:not(.mat-disabled) .mat-slide-toggle-bar {
+    background-color: skyblue;  /*replace with your color*/
```
Comment (roman-fay) [inline src/app/stopwatch/stopwatch.component.css]: copy pasta? :)

**Comment by roman-fay** [comment]
Also - do you handle 12 / 24 hour formats somehow?

File: src/app/services/stopwatch.service.ts (hunk 4 above)
Comment (andriy-korchynskyy) [reply src/app/services/stopwatch.service.ts]: but it's standard. We can not change the number of seconds in an hour. 
Do we also have to move it?

**Pull request closed without merging**